src/LuaEngine/PlayerMethods.h

```diff
@@ -3834,6 +3834,25 @@ namespace LuaPlayer
         return 0;
     }
 
+    /**
+    * Adds a glyph specified by `glyphId` to the [Player]'s current talent specialization into the slot with the index `slotIndex`
+    *
+    * @param uint32 glyphId
+    * @param uint32 slotIndex
+    */
+    int SetGlyph(lua_State* L, Player* player)
+    {
+        uint32 glyphId = Eluna::CHECKVAL<uint32>(L, 2);
+        uint32 slotIndex = Eluna::CHECKVAL<uint32>(L, 3);
+
+        player->SetGlyph(slotIndex, glyphId, !player->GetSession()->PlayerLoading());
+        #if (!defined(TBC) && !defined(CLASSIC))
```

Comment:
The #if  isn't required, since mod-eluna is only for wotlk as inAC 3.3.5.
I'm sure Eluna master would appreciate the expansion dependencies, but we haven't added any to our customisations, so I don't think we should start now.